webpack.config.babel.js

```diff
@@ -4,27 +4,6 @@ import path from 'path';
 const { NODE_ENV } = process.env;
 const production = NODE_ENV === 'production';
 
-const plugins = [
-  new webpack.optimize.OccurenceOrderPlugin(),
-  new webpack.DefinePlugin({
-    'process.env.NODE_ENV': JSON.stringify(NODE_ENV)
-  })
-];
-
-if (production) {
-  plugins.push(
-    new webpack.optimize.UglifyJsPlugin({
```

Comment:
Code optimization and minification is now based on new `mode` property. UglifyJsPlugin is by default used in `production` builds. Running with specified UglifyOptions and with default Webpack configuration yield same effect - no noticeable change in output files size.

If custom Uglify plugin configuration is needed it can be provided using `optimization.minimizer` configuration option.